cookbooks/aws-parallelcluster-install/files/default/ec2_udev_rules/ec2_dev_2_volid.py

```diff
@@ -24,11 +25,27 @@ def get_imdsv2_token():
     return headers
 
 
+def validate_device_name(device_name):
```

Comment:
Is the name of this file correct? `ec2_dev_2_volid` shouldn't be `valid` instead?

Reply:
The name is correct `volid` -> Volume ID.
The script returns the volumen ID given the Block Device Name.

Reply:
ahahah ok thanks